intro changes for smooth migration zeebe user tasks next (#4723)
camunda/camunda-docs: docs/components/concepts/process-instance-migration.md

  - An element that becomes nested in a newly added subprocess
  - An element that was nested in a subprocess is no longer nested in that subprocess
- Mapping instructions cannot change the element type

> - Mapping instructions cannot change the task implementation, e.g. from a job worker user task to a job worker-based user task managed by Camunda, also known as a Camunda user task

Reviewer: ```suggestion
- Mapping instructions cannot change the task implementation, for example, from a job worker user task to a job worker-based user task managed by Camunda (also known as a Camunda user task)
```

Reviewer 2: This fragment is strictly related to [my comment above](https://github.com/camunda/camunda-docs/pull/4723#issuecomment-2548565620). I think it's fine if we just remove the part in partheses:

```suggestion
- Mapping instructions cannot change the task implementation, for example, from a job worker user task to a Camunda user task.
```
